tests/011-test-docker-root-volume-cleanup.sh

```diff
@@ -0,0 +1,59 @@
+source $SRCDIR/libtest.sh
+
+# Test "docker-storage-setup --reset" for DOCKER_ROOT_VOLUME=yes.
+# Returns 0 on success and 1 on failure.
+test_reset_docker_root_volume() {
+  local devs=${TEST_DEVS}
+  local test_status=1
+  local testname=`basename "$0"`
+  local vg_name="dss-test-foo"
+  local mount_filename="var-lib-docker.mount"
+  local docker_root_lv_name="docker-root-lv"
+
+  # Error out if any pre-existing volume group vg named dss-test-foo
+  if vg_exists "$vg_name"; then
+    echo "ERROR: $testname: Volume group $vg_name already exists." >> $LOGS
+    return $test_status
+  fi
+
+  cat << EOF > /etc/sysconfig/docker-storage-setup
+DEVS="$devs"
+VG=$vg_name
+DOCKER_ROOT_VOLUME=yes
+EOF
+
+ # Run docker-storage-setup
+ $DSSBIN >> $LOGS 2>&1
+
+ # Test failed.
+ if [ $? -ne 0 ]; then
+    echo "ERROR: $testname: $DSSBIN Failed." >> $LOGS
+    cleanup $vg_name "$devs"
+    return $test_status
+ fi
+
+  $DSSBIN --reset >> $LOGS 2>&1
+  # Test failed.
+  if [ $? -eq 0 ]; then
+     test_status=0
+     if [ -e "/etc/sysconfig/docker-storage" ] || [ -e "/etc/systemd/system/${mount_filename}" ]; then
+          test_status=1
+     fi
+     if lv_exists "$vg_name" "$docker_root_lv_name";then
+          test_status=1
+     fi
+  fi
+  if [ ${test_status} -eq 1 ]; then
+     echo "ERROR: $testname: $DSSBIN --reset Failed." >> $LOGS
+  fi
+
+  cleanup $vg_name "$devs"
+  return $test_status
```

Comment:
if dss --reset failed, then we have to do the cleanup ourselves. That is remove docker-root-lv, unmount, remove mount unit file etc.